keras_nlp/models/MODEL_CONTRIBUTION_GUIDE.md

```diff
@@ -0,0 +1,204 @@
+from scratch](https://github.com/keras-team/keras-nlp/tree/v0.4.1/keras_nlp/models/distil_bert).
+
+On the other hand, if the model has a small tweak, something simpler can be done.
+For instance, in the Whisper model, the self-attention and cross-attention mechanism
+is exactly the same as vanilla attention, with the exception that the key projection
+layer does not have a bias term. In this case, we can inherit the custom layer
+from one of the standard layers and make minor modifications. See [this PR](https://github.com/keras-team/keras-nlp/pull/801/files#diff-8533ae3a7755c0dbe95ccbb71f85c677297f687bf3884fadefc64f1d0fdce51aR22) for
+more details.
+
+## Steps
+
+To make a complete model contribution, three PRs have to be opened:
+
+1) "Add `DistilBertBackbone`" (with a Colab link demonstrating correct model implementation and weight conversion)
+2) "Add `DistilBertTokenizer` and `DistilBertPreprocessor`"
+3) "Add `DistilBertBackbone` Presets"
+
+
+### Open an issue/Claim an open issue
+Before getting started with the code, it's important to check if there are any
+[open issues](https://github.com/keras-team/keras-nlp/issues?q=is%3Aissue+is%3Aopen+label%3Amodel-contribution)
+related to the model you wish to contribute. If there aren't any open issues,
+you can create one by clicking the "New Issue" button on our repository page. If
+there is an open issue, you can claim it by commenting on the issue and letting
+us know that you're interested in working on it. This helps us keep track of who
+is working on what and avoid duplicated effort.
+
+In case you open a new issue, please follow this template:
+
+-----
+I would like to contribute the DistilBERT model. The DistilBERT model is a more
+efficient, distilled version of BERT; it is 60% smaller than BERT, and 40% faster.
+It is trained with BERT in a student-teacher fashion. At the same time, it does
+not lose much of BERT's performance on the GLUE benchmark. Because DistilBERT is
+efficient and performs well on common downstream tasks, it is widely used in the
+industry and hence, will be a good addition to KerasNLP's model library!
+
+**Will any non-standard layers be needed?**
+
+No, non-standard layers will not be needed. We can use the native
+`keras.layers.TokenAndPositionEmbedding` and `keras_nlp.layers.TransformerEncoder`
+layers since the architecture is the same as BERT's.
+
+**Links**
+- Paper: [DistilBERT, a distilled version of BERT: smaller, faster, cheaper and lighter](https://arxiv.org/abs/1910.01108)
+- Official Repository: https://github.com/huggingface/transformers/tree/v4.26.1/src/transformers/models/distilbert
+-----
+
+<br/>
+
+Note that you need not have all the answers or complete knowledge of the inner
+workings of the model at the time of opening the issue. But it is appreciated if
+you can furnish as much detail as possible to enable us to help you with the
+contribution! 🙂
+
+Occasionally, if the model requires, say a variant of the original self-attention,
```

Comment:
This feels kind of repetitive with what is above, I wonder if we can restructure the flow a bit.